src/integration/pages/storyPage/hausa/__snapshots__/canonical.test.js.snap

```diff
@@ -128,7 +128,7 @@ exports[`Canonical Story Page Image should match snapshot 1`] = `
 <noscript
   class="bbc-17whsh0 edwq9ck0"
 >
-  &lt;strong class="bbc-1l7spgt edwq9ck1"&gt;Na&#x27;urarku na da matsalar sauraren sauti&lt;/strong&gt;
```

Comment:
Out of pure curiosity, I google translated what this meant 😆 😆 😆 😆 
![Screenshot 2024-09-09 at 10 58 39](https://github.com/user-attachments/assets/77edeba3-826d-481d-be2f-5580e600acea)